doc/api/deprecations.md

```diff
@@ -1780,18 +1780,23 @@ cause a lot of issues. See <https://github.com/nodejs/node/issues/14328>.
 ### DEP0089: require('assert')
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28892
+    description: Deprecation revoked.
   - version:
       - v9.9.0
       - v10.0.0
     pr-url: https://github.com/nodejs/node/pull/17002
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Deprecation revoked
 
-Importing assert directly is not recommended as the exposed functions will use
+Importing assert directly was not recommended as the exposed functions use
 loose equality checks. Use `require('assert').strict` instead. The API is the
```

Comment:
Does it make sense to remove the "Use `require('assert').strict` instead" sentence and the next one?

Reply:
Yes. Thanks.